changelog.d/12170.bugfix

```diff
@@ -0,0 +1,3 @@
+Purge a room completely when all local users have forgotten it.
+
+Contributed by @pingiun.
```

Comment:
```suggestion
Purge a room completely when all local users have forgotten it. Contributed by @pingiun.
```

This file gets merged into the changelog, which expects entries to be on one line.